src/CoreMetadata/page.less

```diff
@@ -1,18 +1,35 @@
 .core-metadata-page {
   padding: 20px 20px;
+  box-sizing: content-box;
+  overflow: auto;
+  height: auto;
 }
 
-.core-metadata-page__column {
-  float: left;
-  width: 50%;
+.core-metadata-page__grid { 
+  display: grid; 
+  grid-template-columns: repeat(2, 1fr); 
+  grid-auto-rows: auto;
+} 
+
+.core-metadata-page__box1 { 
```

Comment:
picture, header, table instead of box1, box2, box3 would be easier to understand